Sources/Orbit/Components/HorizontalScroll.swift

```diff
@@ -0,0 +1,235 @@
+import SwiftUI
+
+public enum HorizontalScrollItemWidth {
+    /// Each item will use its own intrinsic width.
+    case intrinsic
+    /// Width ratio compared to container width. Default value is `0.8`.
+    case ratio(CGFloat = 0.8)
+    /// Custom fixed width.
+    case custom(CGFloat)
+}
+
+public enum HorizontalScrollItemHeight {
+    /// Each item will use its intrinsic height. If the item is expanding vertically, its height will be capped to tallest item.
+    case intrinsic
+    /// Custom fixed height.
+    case custom(CGFloat)
+}
+
+/// Groups items onto one accessible row even on small screens.
+///
+/// Can be used to present similar items (such as cards with baggage options) as a single row even on small screens.
+///
+/// - Related components:
+///   - ``ChoiceTile``
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/layout/horizontalscroll/)
+public struct HorizontalScroll<Content: View>: View {
+
+    let spacing: CGFloat
+    let itemWidth: HorizontalScrollItemWidth
+    let itemHeight: HorizontalScrollItemHeight
+    let maxItemWidth: CGFloat?
+    let minHeight: CGFloat?
+    let horizontalPadding: CGFloat
+    let verticalPadding: CGFloat
+    let content: () -> Content
+
+    public var body: some View {
+        SingleAxisGeometryReader(axis: .horizontal) { width in
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(alignment: .top, spacing: spacing) {
+                    content()
+                        .frame(width: itemWidth(forContentWidth: width), alignment: .leading)
+                        .frame(maxWidth: maxItemWidth, alignment: .leading)
+                        .frame(height: resolvedItemHeight, alignment: .top)
+                    
+                    // Provide reliable minimal height
+                    Color.clear
+                        .frame(width: 0, height: minHeight)
```

Comment:
This is at least the third time I see this technique - worth a utility view?

Reply:
Good idea, added a `Strut` component for this purpose.